spec/plans/provision.fmf

```diff
@@ -155,7 +155,13 @@ example: |
 
         # Disk group used to allow possible future extensions
         disk:
-            space: 500 GB
+          - space: 500 GB
+
+        # Multiple disks can be requested as well
+        disk:
+          - space: 2 GB
+          - space: 20 GB
+            type: ssd
```

Comment:
I'd drop the `type:` from the example - unlike the rest of the paragraph, support for `type:` is on the table (yet). "You can specify more than one disk size" is a much smaller promise than "... and pick what type the storage is." I'd prefer extending the specification with `type` later, possibly with a small set of recognized values to start with.

Reply:
Makes sense. Removed. Btw, looking at the config example in #893 I see `size` is used for specifying disk size. Which now sounds even more natural to me. Was there any special motivation behind using `space` instead? Is this naming consistent with some other tools? Otherwise I would personally vote to use `size` which is (probably) more straightforward.

Reply:
Hm, that's true, `space` is a bit odd. Changing it to `size` will require me to change a couple of config files and code, but I'd say it's worth it. +1 for `size` from me.

I think `space` might have originated in Beaker? Not sure though.

Reply:
Both addressed.